drivers/hmis/app/graphql/types/hmis_schema/client.rb

```diff
@@ -339,9 +339,7 @@ def alerts
     def hud_chronic
       return unless current_permission?(permission: :can_view_hud_chronic_status, entity: object)
 
-      # causes n+1 queries
-      enrollments = object.enrollments.where(data_source_id: current_user.hmis_data_source_id)
-      !!object.destination_client&.as_warehouse&.hud_chronic?(scope: enrollments)
+      load_ar_association(object, :enrollments_with_chronic_homelessness).any?
```

Comment:
The frontend tooltip states `Whether this client is considered chronically homeless, as of today. Follows the HUD definition for Chronic at PIT.`

I think to make that continue to be true, we need to:
1) Exclude Exited enrollments. If I understand the ch_enrollment code correctly, this will return `true` if the client has _Ever_ been considered CH at the time of exit. That is not in line with the tooltip description.
2) Return null if client has no open enrollments? Looking at ch_enrollment, it seems like we are unable to calculate whether the client is chronic "today" if they have no open enrollments. But maybe I'm getting the HUD definition of chronic wrong. @eanders do you know whether a client with no open enrollments should have a HUD chronicity status? It seems like we wouldn't have enough information to calculate that.

Reply:
Thanks! I should have thought to check the tooltip to get a better understanding of the intention. 
* I'll limit to chronic status on currently open enrollments (open_excluding_wip)
* I'll add that to the field description in graphql
* I'll assume that this should be false if there are no open enrollments

Reply:
Thanks, I _think_ that's correct, based on my reading of the chronicity calculations.

Reply:
@gigxz I believe you are correct.  `ChEnrollment` is looking at if the client was chronic at entry, or point-in-time for homeless enrollments, but if the enrollment is exited, it would not contribute to the client being "chronic."  We should exclude exited enrollments from this calculation, and maybe re-word the tooltip to `Whether this client has an open enrollment that meets the HUD definition of Chronic at Project Start or Chronic at a Point-in-Time as of today.`